ban/core/models.py

```diff
@@ -117,6 +117,7 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
+    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
```

Comment:
be careful w/ INSEE code, better regexp provided as #257 : /^(2[AB]|[0-9]{2})[0-9]{3}$/
else, w/ CEA_FORMAT regexp, it will accept also 1A001, A1001, AA001, etc ... no ?